docs/connections.md

```diff
@@ -14,15 +14,24 @@ To do so simply add the GCS path to the environment i.e
 ## Using GCP Secret Manager
 DVT supports [Google Cloud Secret Manager](https://cloud.google.com/secret-manager) for storing and referencing secrets in your connection configuration.
 
-If the secret-manager flags are present, the remaining connection flags should reference secret names instead of the secret itself. For example,
-the following BigQuery connection references a secret with name 'dvt-project-id' stored in project MY-PROJECT.
+If the secret-manager flags are present, any of the remaining connection flags can reference secret names instead of the secret itself.
```

Comment:
I think we should add an example mixing secret names and hardcoded values   to make it clear that's a possibility. Not sure if the text change only can clarify that